File: task/src/main/java/com/goexp/galgame/data/util/DateRangeUtil.java
package com.goexp.galgame.data.util;


import com.goexp.galgame.common.model.DateRange;

import java.time.LocalDate;

public class DateRangeUtil {

    public static DateRange yearToNow() {
        return new DateRange(LocalDate.now().withDayOfYear(1), LocalDate.now());
    }

    public static DateRange monthToNow() {
        return new DateRange(LocalDate.now().withDayOfMonth(1), LocalDate.now());
    }
}

File: task/src/main/scala/com/goexp/galgame/data/util/DateRangeUtil.scala
package com.goexp.galgame.data.util

import java.time.LocalDate

import com.goexp.galgame.common.model.DateRange

object DateRangeUtil {
  def yearToNow = new DateRange(LocalDate.now.withDayOfYear(1), LocalDate.now)

  def monthToNow = new DateRange(LocalDate.now.withDayOfMonth(1), LocalDate.now)
}